packages/html-manager/src/libembed-amd.ts

```diff
@@ -4,11 +4,14 @@
 import * as libembed from './libembed';
 
 let cdn = 'https://unpkg.com/';
+let onlyCDN = false;
 
 // find the data-cdn for any script tag, assuming it is only used for embed-amd.js
 const scripts = document.getElementsByTagName('script');
 Array.prototype.forEach.call(scripts, (script: HTMLScriptElement) => {
   cdn = script.getAttribute('data-jupyter-widgets-cdn') || cdn;
+  onlyCDN =
+    onlyCDN || script.getAttribute('data-jupyter-widgets-cdn-only') != null;
```

Comment:
Can you use `hasAttribute` instead? https://developer.mozilla.org/en-US/docs/Web/API/Element/hasAttribute

Reply:
TIL, Thanks!